File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/models/response/ResponseFolder.java
package app.editors.manager.mvp.models.response;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import app.editors.manager.mvp.models.base.Base;
import app.editors.manager.mvp.models.explorer.CloudFolder;

public class ResponseFolder extends Base {

    @SerializedName(KEY_RESPONSE)
    @Expose
    private CloudFolder response;

    public CloudFolder getResponse() {
        return response;
    }

    public void setResponse(CloudFolder response) {
        this.response = response;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseFolder.kt
package app.documents.core.network.manager.models.response

import app.documents.core.network.common.models.BaseResponse
import app.documents.core.network.manager.models.explorer.CloudFolder
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ResponseFolder(
    @SerializedName(KEY_RESPONSE)
    @Expose
    var response: CloudFolder? = null
) : BaseResponse()